lib/osf-components/addon/components/carousel/x-item/component.ts

```diff
@@ -0,0 +1,39 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import { layout, requiredAction } from 'ember-osf-web/decorators/component';
+import template from './template';
+
+@layout(template)
+@tagName('li')
+export default class CarouselItem extends Component {
+    classNameBindings = ['isActive:active', 'slidingIn:slide-in', 'slidingOut:slide-out', 'from'];
```

Comment:
We shouldn't use `classNameBindings`. Also, none of these classes appear to be defined anywhere AFAICT.